### PR TITLE
Feature/take video preview

### PR DESCRIPTION
**Pull request by MutazAshhab**
## Demo

https://user-images.githubusercontent.com/86769131/194013879-94786830-95c8-46e8-9029-a0dbaa4d5fd3.mp4

### OS

- [x] Linux
- [ ] MacOS
- [ ] Windows

### CODE REVIEW

File: src/renderer/components/Editor/TakeGroupComponent.tsx [hunk 1]
```diff
@@ -95,8 +100,15 @@ const TakeGroupComponent = ({
...
   };
 
-  const clickAway = () => {
-    if (!isFirstTimeOpen) {
+  const onClickAway = (event: Event) => {
+    const isInTranscriptionContent = isEventInElement(
+      event,
+      transcriptionContentId
+    );
+    const isInTakeGroup = isEventInElement(event, takeGroupId);
```
Comment (Mhudlark) [inline src/renderer/components/Editor/TakeGroupComponent.tsx:108]: What will happen here if the user clicks away from the current take group onto another take group? Do the id's need to be unique?

**Comment by Mhudlark** [comment]
During testing I noticed a couple of bugs.
1. The take video preview cuts off the end of the last word. If I select a take and then play the entire transcription the word will not be cut off but when I play just that take in the preview, it will be.
2. Clicking on a word in the take video preview (to seek to that time) doesn't seem to work

File: src/renderer/components/Editor/TakeGroupComponent.tsx (hunk 1 above)
Comment (MutazAshhab) [reply src/renderer/components/Editor/TakeGroupComponent.tsx:108]: The ID's dont need to be unique. Clicking on another take group will simply open it

Comment (Mhudlark) [reply src/renderer/components/Editor/TakeGroupComponent.tsx:108]: Is that the behaviour we want? Shouldn't clicking on another take group close the current one? @emmapainter

Comment (emmapainter) [reply src/renderer/components/Editor/TakeGroupComponent.tsx:108]: Yeah @exquisitebin is assigned to "Clicking away from take group closes take group and chooses currently selected take" - so clicking another take group should close the current one

Comment (exquisitebin) [reply src/renderer/components/Editor/TakeGroupComponent.tsx:108]: Thats correct. Clicking on another Take Group closes the currently opened one. That functionality is in develop.

**Review by exquisitebin — APPROVED** [review]
Tested locally and seems to be working well. Good job!

**Pull request closed without merging**